scripts/setup-ubuntu.sh

```diff
@@ -156,12 +156,43 @@ function install_conda {
 
 function install_cuda {
   # See https://developer.nvidia.com/cuda-downloads
+  # if ! dpkg -l cuda-keyring 1>/dev/null; then
+  #   wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/cuda-keyring_1.1-1_all.deb
+  #   $SUDO dpkg -i cuda-keyring_1.1-1_all.deb
+  #   rm cuda-keyring_1.1-1_all.deb
+  #   $SUDO apt update
+  # fi
+  local arch=$(uname -m)
+  local os_ver
+
+  if [[ ${VERSION} =~ "22.04" ]]; then
+    os_ver="ubuntu2204"
+  elif [[ ${VERSION} =~ "24.04" ]]; then
+    os_ver="ubuntu2404"
+  elif [[ ${VERSION} =~ "20.04" ]]; then
```

Comment:
Let’s keep these in order. Probably 24.04, 22.04, 20.04 is best.

Also note that Ubuntu 20.04 is no longer receiving security updates. I recommend that Velox should drop it completely if there are no constraints preventing that.

Reply:
updated in new commit.  Please help review again. 

This is not the only place to deal with Ubuntu versions.   It may handle it in a separate issue.  20.04 is rarely used  I believe. However, it may discuss more if droppint it at all(saying 20.04 is not supported) or a default behavior.